Bottom: 8b4a63e028a035d6fcd54079df892dc6e0f473ab
Top:    6365d78cf7beb5f41bf580cf41d3490002f0955b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-03 22:43:15 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 727786a..2b90e70 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -295,6 +295,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
